khelo/lib/ui/flow/settings/support/components/attachment_item.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:io';
+
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:khelo/domain/extensions/string_extensions.dart';
+import 'package:style/button/action_button.dart';
+import 'package:style/extensions/context_extensions.dart';
+import 'package:style/indicator/progress_indicator.dart';
+import 'package:style/text/app_text_style.dart';
+
+class AttachmentsTypeView extends StatelessWidget {
+  final double height;
+  final double width;
+  final String path;
+  final double borderRadius;
+  final bool isNetwork;
+
+  const AttachmentsTypeView({
+    super.key,
+    required this.path,
+    this.height = 50,
+    this.width = 50,
+    this.borderRadius = 12,
+    this.isNetwork = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: height,
+      width: width,
+      decoration: BoxDecoration(
+        color: context.colorScheme.containerNormal,
+        borderRadius: BorderRadius.circular(borderRadius),
+      ),
+      child: _getFileView(
+        iconColor: context.colorScheme.textDisabled,
+        path: path,
+      ),
+    );
+  }
+
+  Widget _getFileView({
+    required Color iconColor,
+    required String path,
+  }) {
+    if (path.attachmentType.isImage) {
+      return Hero(
+        tag: 'chat-widget/$path',
```

Comment:
do we need it? consider to remove if not needed, consider to give meaningful tag name if needed.